charts/clickhouse/templates/clickhouse-instance/clickhouse-instance.yaml

```diff
@@ -66,7 +66,7 @@ spec:
                 -->
                 <keep_free_space_bytes>{{ .Values.coldStorage.defaultKeepFreeSpaceBytes }}</keep_free_space_bytes>
               </default>
-
+              {{- if eq .Values.coldStorage.type "s3" }}
```

Comment:
There is no `type` in coldStorage? Did you test the change with GCS/S3?

Reply:
done